docs/admin/clustering/scale/demand.md

```diff
@@ -0,0 +1,195 @@
+
+## About
+The approach described below explains how to achieve this using [shard allocation filtering](https://crate.io/docs/crate/reference/en/5.1/general/ddl/shard-allocation.html) on a table that is partitioned by day, the idea is that the period of high activity can be foreseen, so the scaling up will take place the day before a big event, and the scaling down someday after the event has ended.
+
+This same approach can be applied to multiple tables. It is particularly relevant for the larger tables, and smaller tables can be kept on the baseline nodes, but it is always good to consider the impact on querying performance if the small tables will be queried during the big event `JOIN`ed to the big tables that will have data on the temporary nodes.
+
+## Preparing the test environment
+
+In this example, we will imagine that the surge in demand we are preparing for is related to the 2022 FIFA Men’s World Cup running from 20/11/2022 to 18/12/2022.
+
+We will start with a 3 nodes cluster, on which we will create a test table and populate it with some pre-World Cup data:
+
+```sql
+CREATE TABLE test (
+  ts TIMESTAMP,
+  recorddetails TEXT,
+  "day" GENERATED ALWAYS AS date_trunc('day',ts)
+)
+PARTITIONED BY ("day")
+CLUSTERED INTO 4 SHARDS
+WITH (number_of_replicas = 1);
+
+INSERT INTO test (ts) VALUES ('2022-11-18'), ('2022-11-19');
+```
+
+The shards will initially look like this:
+![image|690x167](https://global.discourse-cdn.com/flex020/uploads/crate/original/1X/ac18a9cb507201d8e54771e320501f4aaac0eb16.png)
+
+## Before deploying extra nodes
+
+We want to make sure that the addition of the temporary nodes does not result on data from the large tables getting rebalanced to use these nodes.
+
+We will be able to identify the new nodes by using a custom attribute (`node.attr.storage=temporarynodes`) (see further down for details on how to configure this), so the first step is to configure the existing partitions so that they do not consider the new nodes as suitable targets for shard allocation.
+
+In CrateDB 5.1.2 or higher, you can achieve this with:
+```sql
+/* this applies the setting to all existing partitions and new partitions */
+ALTER TABLE test SET ("routing.allocation.exclude.storage" = 'temporarynodes');
+
+/* then we run this other command so that the setting does not apply to new partitions */
+ALTER TABLE ONLY test RESET ("routing.allocation.exclude.storage");
+```
+In CrateDB 5.9.0 or higher, you don't need to change shard allocations for each
+individual table any longer. Instead, default allocation rules can be set on
+the cluster level:
+> [CrateDB 5.9.0] added support to override `routing.allocation.*` cluster
+> settings with a `routing.allocation.*` table setting. This can be used to
+> define the default routing behavior for all tables with a cluster setting
+> and reroute individual tables by assigning the table setting using `ALTER
+> TABLE SET`.
```

Comment:
@hammerhead suggested to educate readers about the improvements in CrateDB 5.9.0. Thanks!
Is it sensible to come up with example SQL statements here, to emphasize and clarify how a corresponding procedure would work?

NB: With new design elements in the documentation, we have a few more possibilites to convey information in a denser way, or optionally hide it from the main reading flow by using collapsibles. So, don't hesitate to increase the volume of documentation when relaying suggestions. Thanks!

Reply:
If you are fine with the notice, it will be all good. In this case, please acknowledge.
Otherwise, if you have a few examples at hand, and think adding them would be good, just slap them into a comment, and I will add them to the document at this spot.